supreme_court_predictions/data/statistics/DataCleaner.py

```diff
@@ -0,0 +1,224 @@
+"""
+This class is used to clean the data and convert it to a usable format
+"""
+
+# Global Imports
+import os
+import json
+import pandas as pd
```

Comment:
`pandas` isn't in the `pyproject.toml` file, so you'll need to run `poetry add pandas` from your terminal. It seems like it's reading from your computers base python interpreter.

Reply:
I added it to the PR, but it's just something to note for the future.